### PR TITLE
Removed extra unused parameters and added storage account name parameter for create 

### PR DESCRIPTION
**Pull request by ptgadmin-cse**

### CODE REVIEW

File: wvd-templates/Create and provision WVD host pool/mainTemplate.json [hunk 1]
```diff
@@ -98,6 +98,13 @@
                 "description": "True indicating you would like to use managed disks or false indicating you would like to use unmanaged disks."
             }
         },
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "The Name of the Storage Account(Unique Name) to store unamanged disks for unmanaged gallery VM"
```
Comment (ChristianMontoya) [inline wvd-templates/Create and provision WVD host pool/mainTemplate.json]: Change this to be:
"description": "(Required when rdshImageSource = Gallery and RdshUseManagedDisks = False) The name of the storage account to store the unmanaged disks from an Azure Gallery image. If you decide to use unmanaged disks from a custom vhd, the disks will be stored in the same storage account as the image."

Also, please update the ReadMe with the same description.

File: wvd-templates/Update existing WVD host pool/mainTemplate.json [hunk 2]
```diff
@@ -148,93 +141,100 @@
...
-            "type": "String",
+            "type": "string",
             "metadata": {
                 "description": "(Required when rdshUseManagedDisks = False) The resource group containing the storage account of the image vhd file."
             }
         },
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "The Name of the Storage Account(Unique Name) to store unamanged disks for unmanaged gallery VM"
```
Comment (ChristianMontoya) [inline wvd-templates/Update existing WVD host pool/mainTemplate.json]: Please use the same description that I provided in the "Create and provision..." review.

File: wvd-templates/Update existing WVD host pool/mainTemplate.json [hunk 3]
```diff
@@ -148,93 +141,100 @@
...
-            "type": "String",
+            "type": "string",
             "metadata": {
                 "description": "Specifies an organizational unit (OU) for the domain account. Example: OU=testOU;DC=domain;DC=Domain;DC=com"
             }
         },
         "_artifactsLocation": {
-            "defaultValue": "https://raw.githubusercontent.com/Azure/RDS-Templates/staging/wvd-templates/Update%20existing%20WVD%20host%20pool/",
-            "type": "String",
+            "defaultValue": "https://raw.githubusercontent.com/Azure/RDS-Templates/ptg-alignments/wvd-templates/Update%20existing%20WVD%20host%20pool",
```
Comment (ChristianMontoya) [inline wvd-templates/Update existing WVD host pool/mainTemplate.json]: Please change back to "...RDS-Templats/staging/wvd-templates/..."

File: wvd-templates/Create and provision WVD host pool/mainTemplate.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
             "metadata": {
                 "description": "The base URI where artifacts required by this template are located."
             },
-            "defaultValue": "https://raw.githubusercontent.com/Azure/RDS-Templates/staging/wvd-templates/Create%20and%20provision%20WVD%20host%20pool"
+            "defaultValue": "https://raw.githubusercontent.com/Azure/RDS-Templates/ptg-alignments/wvd-templates/Create%20and%20provision%20WVD%20host%20pool"
```
Comment (ChristianMontoya) [inline wvd-templates/Create and provision WVD host pool/mainTemplate.json]: Please rename back to ".../RDS-Templates/staging/wvd-templates/..."

**Comment by ptgadmin-cse** [comment]
Necessary changes are made to the templates, It's ready for your review and merge

**Pull request merged**